includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -993,11 +1003,7 @@ public function get_upe_available_payment_methods() {
 			)
 		);
 
-		$methods_with_fees = array_values( array_intersect( $methods, array_keys( $fees ) ) );
-
-		$methods_with_fees[] = Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
-
-		return $methods_with_fees;
+		return array_values( array_intersect( $methods, array_keys( $fees ) ) );
```

Comment:
I believe it's connected to the previous change, but I'm not following it. Can you enlighten me, please?

Reply:
`$methods_with_fees[] = Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID;` was added before adding fees for Link payment method. So, after fees were added, 'link' was added two times.